src/views/preview/project-view.jsx

```diff
@@ -917,48 +920,51 @@ class Preview extends React.Component {
-                                authorId={this.props.authorId}
-                                authorThumbnailUrl={this.props.authorThumbnailUrl}
-                                authorUsername={this.props.authorUsername}
-                                backpackHost={this.props.backpackHost}
-                                backpackVisible={this.props.canUseBackpack}
-                                basePath="/"
-                                canCreateCopy={this.props.canCreateCopy}
-                                canCreateNew={this.props.canCreateNew}
-                                canEditTitle={this.props.canEditTitleInEditor}
-                                canRemix={this.props.canRemix}
-                                canSave={this.props.canSave}
-                                canShare={this.props.canShare}
-                                className="gui"
-                                cloudHost={this.props.cloudHost}
-                                enableCommunity={this.props.enableCommunity}
-                                hasCloudPermission={this.props.isScratcher}
-                                isShared={this.props.isShared}
-                                isTotallyNormal={this.props.isTotallyNormal}
-                                projectHost={this.props.projectHost}
-                                projectToken={this.props.projectInfo.project_token}
-                                projectId={this.state.projectId}
-                                projectTitle={this.props.projectInfo.title}
-                                renderLogin={this.renderLogin}
-                                onClickLogo={this.handleClickLogo}
-                                onGreenFlag={this.handleGreenFlag}
-                                onLogOut={this.props.handleLogOut}
-                                onOpenRegistration={this.props.handleOpenRegistration}
-                                onProjectLoaded={this.handleProjectLoaded}
-                                onRemixing={this.handleIsRemixing}
-                                onSetLanguage={this.handleSetLanguage}
-                                onShare={this.handleShare}
-                                onToggleLoginOpen={this.props.handleToggleLoginOpen}
-                                onUpdateProjectData={this.handleUpdateProjectData}
-                                onUpdateProjectId={this.handleUpdateProjectId}
-                                onUpdateProjectThumbnail={this.props.handleUpdateProjectThumbnail}
-                                onUpdateProjectTitle={this.handleUpdateProjectTitle}
-                                user={this.props.user}
-                                permissions={this.props.permissions}
-                                onActivateDeck={this.props.onActivateDeck}
-                            />
+                            <>
+                                <TermsOfServiceModal />
```

Comment:
I imagined that we would put `TermsOfServiceModal` inside `IntlGUIWithProjectHandler` as the component already acts as a wrapper to gui.